regtests/ww3_tp2.17/input/ww3_shel.inp

```diff
@@ -37,7 +37,7 @@ $             2 : Last process is reserved for all output, and does no
 $                 computing.
 $             3 : Multiple dedicated output processes.
 $
-   3
+   0 
```

Comment:
Please revert regression test changes or explain why these tests are changing.

Reply:
The PDLIB option only works with option zero, so we have changed them to zero as shown here. It cannot be separated.

Reply:
done